optimism.tokenlist.json

```diff
@@ -1367,9 +1367,25 @@
       "name": "Equalizer",
       "symbol": "EQZ",
       "decimals": 18,
-      "logoURI": "https://ethereum-optimism.github.io/logos/EQZ.png",
+      "logoURI": "https://ethereum-optimism.github.io/logos/EQZ.png"
+    },
+    {
+      "chainId": 42,
+      "address": "0xC99Fc8E8852e5BaB249174a983688c7F28065fB7",
+      "name": "KIDDO",
+      "symbol": "KIDDO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/KIDDO.png"
+    },
+    {
+      "chainId": 69,
+      "address": "0x0E150963D01e8453B7099a3C032345527A4BAfFD",
+      "name": "KIDDO",
+      "symbol": "KIDDO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/KIDDO.png",
       "extensions": {
-        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+        "optimismBridgeAddress": "0x0CFb46528a7002a7D8877a5F7a69b9AaF1A9058e"
```

Comment:
Just checking you are aware that you are using the BitBTC bridge here and NOT the standard bridge.